### PR TITLE
Same way to show tracks everywhere

### LINKED ISSUE #117: Track display not uniform
The unlockable isn't taken into account
image

### PR DESCRIPTION
**Pull request by Tbaut**
closes #117 

image

### CODE REVIEW

File: src/components/DelegationByAmountConviction.tsx [hunk 1]
```diff
@@ -49,24 +35,15 @@ export const DelegationByAmountConviction = ({
       >
         {trackIds.length === 1 ? (
           trackIds.map((trackId) => (
-            <TrackDisplay
-              key={trackId}
-              trackId={trackId}
-              trackName={trackList[trackId]}
-            />
+            <TrackDisplay key={trackId} trackId={trackId} />
```
Comment (wirednkod) [inline src/components/DelegationByAmountConviction.tsx:38]: Is there a need for a `key`? you already set it in the 1st div of the `TrackDisplay` component

File: src/components/DelegationByAmountConviction.tsx [hunk 2]
```diff
@@ -49,24 +35,15 @@ export const DelegationByAmountConviction = ({
...
-            buttonClassName="w-auto items-beginning"
+            buttonClassName="w-auto"
           >
             {trackIds.sort().map((trackId) => (
-              <TrackDisplay
-                key={trackId}
-                trackId={trackId}
-                trackName={trackList[trackId]}
-              />
+              <TrackDisplay key={trackId} trackId={trackId} />
```
Comment (wirednkod) [inline src/components/DelegationByAmountConviction.tsx:46]: same as above

**Review by wirednkod — APPROVED** [review]

File: src/components/DelegationByAmountConviction.tsx (hunk 1 above)
Comment (Tbaut) [reply src/components/DelegationByAmountConviction.tsx:38]: nope indeed, wrong copy/paste

Comment (Tbaut) [reply src/components/DelegationByAmountConviction.tsx:38]: oh wait, not it's totally right, it's the first element of a map, if you remove it, it'll throw in the console.

File: src/components/DelegationByAmountConviction.tsx (hunk 2 above)
Comment (Tbaut) [reply src/components/DelegationByAmountConviction.tsx:46]: in a map yes

**Pull request merged**